src/mixins/itext_key_behavior.mixin.js

```diff
@@ -51,6 +51,7 @@ fabric.util.object.extend(fabric.IText.prototype, /** @lends fabric.IText.protot
 
   onClick: function() {
     // No need to trigger click event here, focus is enough to have the keyboard appear on Android
+    // 2015-02-05 with forced focus on enterEditing maybe we can remove this now.
```

Comment:
You think it, or are you asking me to remove it?
for me remove. We have plenty of user ready to open another issue to tell us we were wrong.